user_guide.adoc

```diff
@@ -1377,30 +1377,82 @@ library" for expression language.
 |Return type
 |Description
 
-|to_s
+|`to_s`
 |String
 |Converts integer into a string using decimal representation
-
-|to_s(radix)
```

Comment:
Um, you're right, I'm sorry! I guess we need to have a good solid checklist of methods that need to be implemented, some test suite to check conformance, and, ideally, make it somehow in sync with the documentation automatically...

I wonder if it's a good idea to make it auto-generated somehow...